app/controllers/sightings_controller.rb

```diff
@@ -0,0 +1,39 @@
+class SightingsController < ApplicationController
+    def index
+        sightings = Sighting.all
+        render json: sightings
+    end
+    def create
+        sighting = Sighting.create(sighting_params)
+        if sighting.valid?
+          render json: sighting
+        else
+          render json: sighting.errors
+        end
+        
+    end
+
+    def sighting_params
+        params.require(:sighting).permit(:longitude, :date, :latitude, :animal_id)
+      end
```

Comment:
Since these are strong params, we need to place this inside a private section of the controller.